.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Make constitution/non-constitutional change from meeting on YYYY-MM-DD
+
+Description of the changes.
+
+Proposed By Name.
+
+**Attendees**:
+  - Name: Role
```

Comment:
`Role` maybe doesn't make sense anymore since they will all just be 'exec committee member'

Reply:
@jaredkhan At the moment it does. The new structure isn't relevant to this yet.

Reply:
This pull request is before the one for the new structure. We need a future PR for that.

The committee also only voted to allow grammatical changes - not these ones.